lib/players/computer.rb

```diff
@@ -0,0 +1,32 @@
+module Players
+  class Computer < Player
+    def move(board)
+      move = nil
+      # first move, take the middle square
+      if !board.taken?(5)
+        move = "5"
+      # second move, take the top left corner
+      elsif board.turn_count == 1
+        move = "1"
+      # third move, check top right and bottom left corner
+      elsif board.turn_count == 2
+        move = [3, 7].detect{|i| !board.taken?(i)}.to_s
+      # Check for any combinations of 2 matching tokens and an empty square  
+      else
+        Game::WIN_COMBINATIONS.detect do |combo|
+          # Check if you have any chances to win
+          if combo.select{|i| board.position(i+1) == token}.size == 2 && combo.any?{|i| board.position(i+1) == " "}
```

Comment:
Instead of `select` and then querying the size, why not use the count method, similarly to how you used it in the `turn_count` method?